apps/settings/elements/call_cf_settings.html

```diff
@@ -1,16 +1,12 @@
 <element name="call-cfSettings" extends="section">
   <template>
 
-    <header>
-      <button type="reset"><span data-l10n-id="back" class="icon icon-back"></span></button>
+    <gaia-header action="back" data-href="#call">
       <h1 data-l10n-id="callForwarding-header"></h1>
-    </header>
+    </gaia-header>
 
     <div>
       <ul id="list-callForwarding">
-        <li id="fdnSettingsBlocked" class="description" hidden>
```

Comment:
Why this is being removed? We have to display the message and hide the call forwarding menu item when fdn is enabled.